api/controller.py

```diff
@@ -653,7 +654,7 @@ def groups(self, lane_identifier):
             worklist=lane, base_class=FeaturedFacets,
             base_class_constructor_kwargs=facet_class_kwargs
         )
-        annotator = self.manager.annotator(lane)
+        annotator = self.manager.annotator(lane, facets)
```

Comment:
Are you sure this is the only place with entrypoints? What about search?

Reply:
Just added it to the annotator when making a search, thanks. There are more instances of `self = self.manager.annotator` but I don't think they are relevant?